src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -219,9 +220,10 @@ private TemplateInstance templateBodyForSubscriptionType(EmailSubscriptionType s
 
         log.info(String.format("Running %s email aggregation for period (%s, %s)", emailSubscriptionType.toString(), startTime.toString(), endTime.toString()));
 
+        final String bundle = "insights";
```

Comment:
Does it make sense to hardcode this? But it is probably good enough for now

Reply:
I have a PR that removes this (will need to rebase to this one and finish)
https://github.com/RedHatInsights/notifications-backend/pull/108